test/board.spec.js

```diff
@@ -19,6 +19,46 @@ test('creates seven different types of orbs', t => {
     t.same(board.availableTypes, _.range(7));
 });
 
+test('finds all mathes of three, including overlaps', t => {
+    let testMatches = [ 
+        [ [ 0, 1 ], [ 0, 2 ], [ 0, 3 ] ],
+        [ [ 0, 2 ], [ 0, 3 ], [ 0, 4 ] ],
+        [ [ 2, 3 ], [ 3, 3 ], [ 4, 3 ] ],
+        [ [ 3, 2 ], [ 3, 3 ], [ 3, 4 ] ] 
+    ];
+    board.orbs = [ 
+        [ 3, 2, 2, 2, 2, 3, 6, 6 ],
+        [ 4, 2, 0, 5, 3, 0, 1, 5 ],
+        [ 1, 4, 3, 2, 5, 4, 6, 6 ],
+        [ 3, 5, 2, 2, 2, 3, 3, 4 ],
+        [ 1, 1, 3, 2, 5, 2, 6, 2 ],
+        [ 3, 2, 5, 4, 5, 0, 4, 0 ],
+        [ 1, 3, 1, 6, 1, 6, 3, 3 ],
+        [ 1, 0, 3, 0, 1, 1, 3, 3 ] 
+    ];
+    
+    t.same(board.findMatches(), testMatches);
+});
+
+test('combines all matches of 3 to create matches of 4+', t => {
+    let testCombined = [ 
+        [ [ 0, 1 ], [ 0, 2 ], [ 0, 3 ], [ 0, 4 ] ],
+        [ [ 2, 3 ], [ 3, 2 ], [ 3, 3 ], [ 3, 4 ], [ 4, 3 ] ] 
+    ];
+    board.orbs = [ 
+        [ 3, 2, 2, 2, 2, 3, 6, 6 ],
+        [ 4, 2, 0, 5, 3, 0, 1, 5 ],
+        [ 1, 4, 3, 2, 5, 4, 6, 6 ],
+        [ 3, 5, 2, 2, 2, 3, 3, 4 ],
+        [ 1, 1, 3, 2, 5, 2, 6, 2 ],
+        [ 3, 2, 5, 4, 5, 0, 4, 0 ],
+        [ 1, 3, 1, 6, 1, 6, 3, 3 ],
+        [ 1, 0, 3, 0, 1, 1, 3, 3 ] 
+    ];
+    
+    t.same(board.combineMatches(board.findMatches()), testCombined);
```

Comment:
Why are you calling `combineMatches` with an argument? It doesn't appear to accept one.